xml/art_solution_automation.xml

```diff
@@ -166,6 +166,175 @@ https://confluence.suse.com/x/8IEnGw
+          <para>
+            All steps in <xref linkend="pro-sol-autom-hana-apply-salt-formulas"/> must be
+            executed on the &saltmaster; machine.</para>
+        </formalpara>
+      </listitem>
+      <listitem>
+        <formalpara>
+          <title>&salt; Client only</title>
+          <para>
+            All steps in <xref linkend="pro-sol-autom-hana-apply-salt-formulas"/> must be
+            executed in all of the &salt; clients where the formulas are going to executed.
+          </para>
+        </formalpara>
+      </listitem>
+    </itemizedlist>
+
+    <procedure xml:id="pro-sol-autom-hana-apply-salt-formulas">
+      <title>Applying &salt; Formulas</title>
+      <para>
+        In this procedure we show how an example how to create a pillar structure for
+        a system with &hana;, &netweaver;, and with DRBD. All this uses &sleha;.
+      </para>
+      <step>
+        <para>
+          Install the &salt; formulas with <command>zypper</command>.
+          Get the package name from the upstream GitHub repository.
+        </para>
+      </step>
+      <step>
+        <para>
+          Create the pillar files structure.
+        </para>
+        <para>
+          The pillar files work as configuration files for the &salt; formulas
+          and are intended as user input. To use the files, create a
+          <filename>top.sls</filename> file which references other files you
+          want to apply.
+        </para>
+        <substeps>
+          <step>
+            <para>
+              Create the directory <filename class="directory">/srv/pillar</filename>
+              and its subdirectories <filename class="directory">hana</filename>,
+              <filename class="directory">netweaver</filename>,
+              and <filename class="directory">drbd</filename>:
+            </para>
+            <screen>&prompt.root;<command>mkdir</command> --parents /srv/pillar/{hana,netweaver,drbd}</screen>
+            <para>
+              The directory names match with the names in the <filename>top.sls</filename>
+              file (see next step).
+            </para>
+          </step>
+          <step>
+            <para>
+              Create the file <filename>/srv/pillar/top.sls</filename> with the following content:
+            </para>
+            <screen language="sls">base:
+    'hana01,hana02':
+        - hana.hana
+        - hana.cluster
+
+    'netweaver01,netweaver02':
+        - netweaver.netweaver
+        - netweaver.cluster
+
+    'drbd01,drbd02':
+        - drbd.drbd
+        - drbd.cluster</screen>
+          </step>
+        </substeps>
+      </step>
+      <step>
+        <para>
+          Download the pillar files and store them in the correct directory under <filename
+            class="directory">/srv/pillar</filename>:
+        </para>
+        <substeps>
+          <step>
+            <remark>toms 2020-09-23: You can download the files and store them
```

Comment:
I don't know what to say... One or two steps looks the same to me.
Actually, I would just say that the pillar configuration files must be created, and downloading them from the github repositories is one option. In any case.
The current text looks good to me

Reply:
Hmn... :thinking: ... did I get that right, you think the text "Download the pillar files and store them in the correct directory under `/srv/pillar/`" would be enough?

Reply:
What I mean is that the user needs to create the pillar files and put the content. Downloading them from the current examples is one option (maybe the best option by now). I just wanted to emphisize (that they don't necessarily need to download). It think the current text is good.

Reply:
I've tried to rearrange these substeps into steps, I think that makes it clearer. Did that in commit fd9bd33.

Reply:
> Downloading them from the current examples is one option

Ahh, I see. I'm thinking about if I should mention this or better leave it as is (to not complicate things). :thinking: I think, I leave it for the time being.